### ISSUE TITLE
npm start not working

### ISSUE THREAD

**Issue by jazzholicbae** [codevet/daipay#1]
only npm run dev is the working one

tried with npm run build -> npm start however the dist folder was empty

**Comment by canpekdemir1** [comment]
hello 
I'm NEW IN GITHUB
I couldn't find where to write my problem
I get such error on npm startup

root@myhost ~]# cd
[root@myhost ~]# cd home
[root@myhost ~]# cd node
[root@myhost ~]# npm start
npm ERR! Error: ENOENT, open '/root/package.json'
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Linux 3.10.0-1062.9.1.el7.x86_64
npm ERR! command "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! cwd /root
npm ERR! node -v v0.10.30
npm ERR! npm -v 1.4.21
npm ERR! path /root/package.json
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /root/npm-debug.log
npm ERR! not ok code 0
[root@myhost ~]


How to fix this problem

**Comment by defisapiens** [comment]
Fixed by https://github.com/codevet/daipay/commit/e349b459d3917adb6aedcb8733bad317031cc97d

**Comment by jazzholicbae** [comment]
@codevet Still not fixed, it gives me Not Found error

**Comment by defisapiens** [comment]
please provide the log of npm run build

**Comment by jazzholicbae** [comment]
```
ubuntu@ubuntu:~/daipay$ npm run build

> daipay@1.0.1 build /home/ubuntu/daipay
> NODE_ENV=production webpack --config ./tools/app.webpack.config.js --mode production --verbose

WARNING: NODE_ENV value of 'production' did not match any deployment config file names.
WARNING: See https://github.com/lorenwest/node-config/wiki/Strict-Mode
Generating client config
Hash: 0bb1366bbe6e1a0db30b20c9c1f583e2f72551e6
Version: webpack 4.41.5
Child
    Hash: 0bb1366bbe6e1a0db30b
    Time: 3928ms
    Built at: 01/17/2020 4:43:38 PM
    PublicPath: /
                              Asset      Size  Chunks                         Chunk Names
         0.0bb1366bbe6e1a0db30b.css   196 KiB       0  [emitted] [immutable]  main
                         index.html  1.37 KiB          [emitted]
       main.89ba3a633419c20c1e02.js  4.35 KiB       0  [emitted] [immutable]  main
    runtime.0f3a5ec763f36a15c1ea.js  1.46 KiB       1  [emitted] [immutable]  runtime
     vendor.9a42cf3a4b2afbfc6dbc.js  55.6 KiB       2  [emitted] [immutable]  vendor
    Entrypoint main [big] = runtime.0f3a5ec763f36a15c1ea.js vendor.9a42cf3a4b2afbfc6dbc.js 0.0bb1366bbe6e1a0db30b.css main.89ba3a633419c20c1e02.js
    chunk    {0} 0.0bb1366bbe6e1a0db30b.css, main.89ba3a633419c20c1e02.js (main) 203 KiB ={1}= ={2}= [initial] [rendered]
        > main
     [13] ./static/client.json 78 bytes {0} [depth 4] [built]
          [exports: contracts, default]
          [only some exports used: default]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation config [42] ./src/web/main.js + 2 modules 4:0-28
          harmony import specifier config [42] ./src/web/main.js + 2 modules 24:154-160
     [14] multi ./src/web/index.js 28 bytes {0} [depth 0] [built]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          multi entry
     [15] ./src/web/index.js 268 bytes {0} [depth 1] [built]
          [no exports]
          ModuleConcatenation bailout: Module is referenced from these modules with unsupported syntax: multi ./src/web/index.js (referenced with single entry)
          single entry ./src/web/index.js [14] multi ./src/web/index.js main[0]
     [41] ./src/web/main.scss 39 bytes {0} [depth 3] [built]
          [no exports used]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation ./main.scss [42] ./src/web/main.js + 2 modules 5:0-21
     [42] ./src/web/main.js + 2 modules 6.33 KiB {0} [depth 2] [built]
          [exports: default]
          ModuleConcatenation bailout: Cannot concat with ./node_modules/@babel/runtime/helpers/extends.js (<- Module is not an ECMAScript module)
          ModuleConcatenation bailout: Cannot concat with ./node_modules/@hyperapp/router/src/index.js
          ModuleConcatenation bailout: Cannot concat with ./node_modules/hyperapp/src/index.js
          ModuleConcatenation bailout: Cannot concat with ./node_modules/qrcode/lib/browser.js (<- Module is not an ECMAScript module)
          ModuleConcatenation bailout: Cannot concat with ./static/client.json (<- Module is not an ECMAScript module)
          cjs require ./main [15] ./src/web/index.js 4:11-28
          | ./src/web/main.js 1.58 KiB [depth 2] [built]
          |     [exports: default]
          |     ModuleConcatenation bailout: Module is referenced from these modules with unsupported syntax: ./src/web/index.js (referenced with cjs require)
          |     cjs require ./main [15] ./src/web/index.js 4:11-28
          | ./src/web/invoice.js 4.55 KiB [depth 3] [built]
          |     [exports: default]
          |     [all exports used]
          |     harmony side effect evaluation ./invoice  ./src/web/main.js 4:0-32
          |     harmony import specifier ./invoice  ./src/web/main.js 24:11-18
          |     harmony import specifier ./invoice  ./src/web/main.js 28:11-18
          |     harmony import specifier ./invoice  ./src/web/main.js 48:23-30
          | ./src/web/helpers.js 181 bytes [depth 4] [built]
          |     [exports: classNames, displayDAI]
          |     [all exports used]
          |     harmony side effect evaluation ./helpers  ./src/web/invoice.js 2:0-51
          |     harmony import specifier ./helpers  ./src/web/invoice.js 90:17-27
          |     harmony import specifier ./helpers  ./src/web/invoice.js 115:73-83
          |     harmony import specifier ./helpers  ./src/web/invoice.js 116:82-92
          |     harmony import specifier ./helpers  ./src/web/invoice.js 116:188-198
          |     harmony import specifier ./helpers  ./src/web/invoice.js 116:290-300
         + 2 hidden modules
    chunk    {1} runtime.0f3a5ec763f36a15c1ea.js (runtime) 0 bytes ={0}= ={2}= [entry] [rendered]
        > main
    chunk    {2} vendor.9a42cf3a4b2afbfc6dbc.js (vendor) 157 KiB ={0}= ={1}= [initial] [rendered] split chunk (cache group: vendor) (name: vendor)
        > main
      [4] ./node_modules/@hyperapp/router/src/index.js + 6 modules 4.22 KiB {2} [depth 3] [built]
          [exports: Link, Route, Switch, Redirect, location]
          [only some exports used: Route, location]
          ModuleConcatenation bailout: Cannot concat with ./node_modules/hyperapp/src/index.js because of ./src/web/index.js
          harmony side effect evaluation @hyperapp/router [42] ./src/web/main.js + 2 modules 3:0-85
          harmony import specifier @hyperapp/router [42] ./src/web/main.js + 2 modules 23:12-18
          harmony import specifier @hyperapp/router [42] ./src/web/main.js + 2 modules 27:12-18
          harmony import specifier @hyperapp/router [42] ./src/web/main.js + 2 modules 40:7-12
          harmony import specifier @hyperapp/router [42] ./src/web/main.js + 2 modules 43:8-13
          harmony import specifier @hyperapp/router [42] ./src/web/main.js + 2 modules 67:10-16
          |    7 modules
     [27] (webpack)/buildin/global.js 472 bytes {2} [depth 8] [built]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          cjs require global [26] ./node_modules/node-libs-browser/node_modules/buffer/index.js 1:0-47
         + 36 hidden modules

    LOG from webpack.buildChunkGraph.visitModules
    <t> prepare: 0.394158ms
    <t> visiting: 0.431719ms

    WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
    Entrypoints:
      main (258 KiB)
          runtime.0f3a5ec763f36a15c1ea.js
          vendor.9a42cf3a4b2afbfc6dbc.js
          0.0bb1366bbe6e1a0db30b.css
          main.89ba3a633419c20c1e02.js


    WARNING in webpack performance recommendations:
    You can limit the size of your bundles by using import() or require.ensure to lazy load some parts of your application.
    For more info visit https://webpack.js.org/guides/code-splitting/
    Child html-webpack-plugin for "index.html":
        PublicPath: /
         1 asset
        Entrypoint undefined = index.html
        chunk    {0} index.html 536 KiB [entry] [rendered]
            > /home/ubuntu/daipay/node_modules/html-webpack-plugin/lib/loader.js!/home/ubuntu/daipay/node_modules/html-webpack-template/index.ejs
         [2] (webpack)/buildin/global.js 472 bytes {0} [depth 2] [built]
             ModuleConcatenation bailout: Module is not an ECMAScript module
             cjs require global [1] ./node_modules/lodash/lodash.js 1:0-41
         [3] (webpack)/buildin/module.js 497 bytes {0} [depth 2] [built]
             ModuleConcatenation bailout: Module is not an ECMAScript module
             cjs require module [1] ./node_modules/lodash/lodash.js 1:0-41
             + 2 hidden modules

        LOG from webpack.buildChunkGraph.visitModules
        <t> prepare: 0.102329ms
        <t> visiting: 0.046766ms
    Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!node_modules/sass-loader/dist/cjs.js!src/web/main.scss:
        PublicPath: /
        Entrypoint mini-css-extract-plugin = *
        chunk    {0} * (mini-css-extract-plugin) 199 KiB [entry] [rendered]
            > !!/home/ubuntu/daipay/node_modules/css-loader/dist/cjs.js!/home/ubuntu/daipay/node_modules/sass-loader/dist/cjs.js!/home/ubuntu/daipay/src/web/main.scss mini-css-extract-plugin
         [0] ./node_modules/css-loader/dist/cjs.js!./node_modules/sass-loader/dist/cjs.js!./src/web/main.scss 196 KiB {0} [depth 0] [built]
             ModuleConcatenation bailout: Module is not an ECMAScript module
             single entry !!/home/ubuntu/daipay/node_modules/css-loader/dist/cjs.js!/home/ubuntu/daipay/node_modules/sass-loader/dist/cjs.js!/home/ubuntu/daipay/src/web/main.scss  mini-css-extract-plugin
             + 1 hidden module

        LOG from webpack.buildChunkGraph.visitModules
        <t> prepare: 0.01973ms
        <t> visiting: 0.014519ms
Child
    Hash: 20c9c1f583e2f72551e6
    Time: 1995ms
    Built at: 01/17/2020 4:43:36 PM
          Asset      Size  Chunks             Chunk Names
    ./server.js  10.8 KiB       0  [emitted]  main
    Entrypoint main = ./server.js
    chunk    {0} ./server.js (main) 23.9 KiB [entry] [rendered]
        > main
      [0] external "@babel/runtime/regenerator" 42 bytes {0} [depth 3] [built]
          [only some exports used: default]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 1:0-61
          harmony side effect evaluation @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 1:0-61
          harmony side effect evaluation @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 1:0-61
          harmony side effect evaluation @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 1:0-61
          harmony side effect evaluation @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 1:0-61
          harmony import specifier @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 13:2-21
          harmony import specifier @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 15:2-21
          harmony import specifier @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 16:2-21
          harmony import specifier @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 16:11-30
          harmony import specifier @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 18:11-30
          harmony import specifier @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 18:11-30
          harmony import specifier @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 21:6-25
          harmony import specifier @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 23:15-34
          harmony import specifier @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 64:2-21
          harmony import specifier @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 66:11-30
          harmony import specifier @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 125:6-25
          harmony import specifier @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 126:15-34
          harmony import specifier @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 144:4-23
          harmony import specifier @babel/runtime/regenerator [21] ./src/server/index.js + 11 modules 145:13-32
      [1] external "config" 42 bytes {0} [depth 2] [built]
          [only some exports used: default]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation config [21] ./src/server/index.js + 11 modules 2:0-28
          harmony side effect evaluation config [21] ./src/server/index.js + 11 modules 2:0-28
          harmony side effect evaluation config [21] ./src/server/index.js + 11 modules 3:0-28
          harmony side effect evaluation config [21] ./src/server/index.js + 11 modules 5:0-28
          harmony import specifier config [21] ./src/server/index.js + 11 modules 5:11-17
          harmony import specifier config [21] ./src/server/index.js + 11 modules 5:37-43
          harmony side effect evaluation config [21] ./src/server/index.js + 11 modules 6:0-28
          harmony import specifier config [21] ./src/server/index.js + 11 modules 6:10-16
          harmony import specifier config [21] ./src/server/index.js + 11 modules 6:42-48
          harmony side effect evaluation config [21] ./src/server/index.js + 11 modules 7:0-28
          harmony import specifier config [21] ./src/server/index.js + 11 modules 8:54-60
          harmony import specifier config [21] ./src/server/index.js + 11 modules 8:75-81
          harmony import specifier config [21] ./src/server/index.js + 11 modules 23:16-22
          harmony import specifier config [21] ./src/server/index.js + 11 modules 23:33-39
          harmony import specifier config [21] ./src/server/index.js + 11 modules 66:18-24
          harmony import specifier config [21] ./src/server/index.js + 11 modules 73:48-54
      [2] external "@babel/runtime/helpers/asyncToGenerator" 42 bytes {0} [depth 3] [built]
          [only some exports used: default]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation @babel/runtime/helpers/asyncToGenerator [21] ./src/server/index.js + 11 modules 2:0-72
          harmony side effect evaluation @babel/runtime/helpers/asyncToGenerator [21] ./src/server/index.js + 11 modules 2:0-72
          harmony side effect evaluation @babel/runtime/helpers/asyncToGenerator [21] ./src/server/index.js + 11 modules 2:0-72
          harmony side effect evaluation @babel/runtime/helpers/asyncToGenerator [21] ./src/server/index.js + 11 modules 2:0-72
          harmony side effect evaluation @babel/runtime/helpers/asyncToGenerator [21] ./src/server/index.js + 11 modules 2:0-72
          harmony import specifier @babel/runtime/helpers/asyncToGenerator [21] ./src/server/index.js + 11 modules 11:31-48
          harmony import specifier @babel/runtime/helpers/asyncToGenerator [21] ./src/server/index.js + 11 modules 13:26-43
          harmony import specifier @babel/runtime/helpers/asyncToGenerator [21] ./src/server/index.js + 11 modules 14:13-30
          harmony import specifier @babel/runtime/helpers/asyncToGenerator [21] ./src/server/index.js + 11 modules 19:27-44
          harmony import specifier @babel/runtime/helpers/asyncToGenerator [21] ./src/server/index.js + 11 modules 62:14-31
          harmony import specifier @babel/runtime/helpers/asyncToGenerator [21] ./src/server/index.js + 11 modules 123:27-44
          harmony import specifier @babel/runtime/helpers/asyncToGenerator [21] ./src/server/index.js + 11 modules 142:15-32
      [3] external "ethers" 42 bytes {0} [depth 4] [built]
          [only some exports used: ethers]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation ethers [21] ./src/server/index.js + 11 modules 1:0-32
          harmony side effect evaluation ethers [21] ./src/server/index.js + 11 modules 2:0-32
          harmony import specifier ethers [21] ./src/server/index.js + 11 modules 5:21-27
          harmony side effect evaluation ethers [21] ./src/server/index.js + 11 modules 7:0-32
          harmony import specifier ethers [21] ./src/server/index.js + 11 modules 8:21-27
          harmony import specifier ethers [21] ./src/server/index.js + 11 modules 29:25-31
      [4] external "koa-router" 42 bytes {0} [depth 3] [built]
          [only some exports used: default]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation koa-router [21] ./src/server/index.js + 11 modules 3:0-32
          harmony import specifier koa-router [21] ./src/server/index.js + 11 modules 7:17-23
          harmony side effect evaluation koa-router [21] ./src/server/index.js + 11 modules 8:0-32
          harmony import specifier koa-router [21] ./src/server/index.js + 11 modules 12:17-23
      [5] external "mongoose" 42 bytes {0} [depth 6] [built]
          [only some exports used: default]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation mongoose [21] ./src/server/index.js + 11 modules 1:0-32
          harmony import specifier mongoose [21] ./src/server/index.js + 11 modules 10:2-10
          harmony import specifier mongoose [21] ./src/server/index.js + 11 modules 13:2-10
          harmony import specifier mongoose [21] ./src/server/index.js + 11 modules 14:9-17
      [6] external "shortid" 42 bytes {0} [depth 4] [built]
          [only some exports used: default]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation shortid [21] ./src/server/index.js + 11 modules 6:0-30
          harmony import specifier shortid [21] ./src/server/index.js + 11 modules 27:28-35
          harmony import specifier shortid [21] ./src/server/index.js + 11 modules 46:23-30
      [7] external "bignumber.js" 42 bytes {0} [depth 4] [built]
          [only some exports used: default]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation bignumber.js [21] ./src/server/index.js + 11 modules 1:0-37
          harmony import specifier bignumber.js [21] ./src/server/index.js + 11 modules 3:13-22
          harmony import specifier bignumber.js [21] ./src/server/index.js + 11 modules 6:9-18
      [8] external "koa" 42 bytes {0} [depth 3] [built]
          [only some exports used: default]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation koa [21] ./src/server/index.js + 11 modules 6:0-22
          harmony import specifier koa [21] ./src/server/index.js + 11 modules 13:14-17
      [9] external "koa-bodyparser" 42 bytes {0} [depth 3] [built]
          [only some exports used: default]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation koa-bodyparser [21] ./src/server/index.js + 11 modules 9:0-40
          harmony import specifier koa-bodyparser [21] ./src/server/index.js + 11 modules 15:8-18
     [10] external "koa-mount" 42 bytes {0} [depth 3] [built]
          [only some exports used: default]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation koa-mount [21] ./src/server/index.js + 11 modules 10:0-30
          harmony import specifier koa-mount [21] ./src/server/index.js + 11 modules 15:47-52
     [11] external "@babel/runtime/helpers/classCallCheck" 42 bytes {0} [depth 4] [built]
          [only some exports used: default]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation @babel/runtime/helpers/classCallCheck [21] ./src/server/index.js + 11 modules 3:0-68
          harmony import specifier @babel/runtime/helpers/classCallCheck [21] ./src/server/index.js + 11 modules 13:4-19
     [12] external "@babel/runtime/helpers/createClass" 42 bytes {0} [depth 4] [built]
          [only some exports used: default]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation @babel/runtime/helpers/createClass [21] ./src/server/index.js + 11 modules 4:0-62
          harmony import specifier @babel/runtime/helpers/createClass [21] ./src/server/index.js + 11 modules 16:2-14
     [13] external "lodash" 42 bytes {0} [depth 3] [built]
          [only some exports used: default]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation lodash [21] ./src/server/index.js + 11 modules 3:0-23
          harmony import specifier lodash [21] ./src/server/index.js + 11 modules 78:17-18
     [14] ./src/contracts/abi/ERC20.json 1.6 KiB {0} [depth 4] [built]
          [exports: 0, 1, 2, 3, 4, 5, 6, 7, default]
          [only some exports used: default]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation ./abi/ERC20.json [21] ./src/server/index.js + 11 modules 4:0-39
          harmony import specifier ./abi/ERC20.json [21] ./src/server/index.js + 11 modules 5:77-84
     [15] external "request" 42 bytes {0} [depth 3] [built]
          [only some exports used: default]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation request [21] ./src/server/index.js + 11 modules 3:0-30
          harmony import specifier request [21] ./src/server/index.js + 11 modules 34:16-23
     [16] multi ./src/server/index.js 28 bytes {0} [depth 0] [built]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          multi entry
     [17] external "@babel/runtime/helpers/slicedToArray" 42 bytes {0} [depth 3] [built]
          [no exports used]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation @babel/runtime/helpers/slicedToArray [21] ./src/server/index.js + 11 modules 3:0-66
     [18] external "fs" 42 bytes {0} [depth 3] [built]
          [no exports used]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation fs [21] ./src/server/index.js + 11 modules 4:0-20
     [19] external "path" 42 bytes {0} [depth 3] [built]
          [no exports used]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation path [21] ./src/server/index.js + 11 modules 5:0-24
     [20] external "koa-logger" 42 bytes {0} [depth 3] [built]
          [no exports used]
          ModuleConcatenation bailout: Module is not an ECMAScript module
          harmony side effect evaluation koa-logger [21] ./src/server/index.js + 11 modules 7:0-35
     [21] ./src/server/index.js + 11 modules 21.5 KiB {0} [depth 1] [built]
          [no exports]
          ModuleConcatenation bailout: Cannot concat with ./src/contracts/abi/ERC20.json (<- Module is not an ECMAScript module)
          ModuleConcatenation bailout: Cannot concat with external "@babel/runtime/helpers/asyncToGenerator" (<- Module is not an ECMAScript module)
          ModuleConcatenation bailout: Cannot concat with external "@babel/runtime/helpers/classCallCheck" (<- Module is not an ECMAScript module)
          ModuleConcatenation bailout: Cannot concat with external "@babel/runtime/helpers/createClass" (<- Module is not an ECMAScript module)
          ModuleConcatenation bailout: Cannot concat with external "@babel/runtime/regenerator" (<- Module is not an ECMAScript module)
          ModuleConcatenation bailout: Cannot concat with external "bignumber.js" (<- Module is not an ECMAScript module)
          ModuleConcatenation bailout: Cannot concat with external "config" (<- Module is not an ECMAScript module)
          ModuleConcatenation bailout: Cannot concat with external "ethers" (<- Module is not an ECMAScript module)
          ModuleConcatenation bailout: Cannot concat with external "koa" (<- Module is not an ECMAScript module)
          ModuleConcatenation bailout: Cannot concat with external "koa-bodyparser" (<- Module is not an ECMAScript module)
          ModuleConcatenation bailout: Cannot concat with external "koa-mount" (<- Module is not an ECMAScript module)
          ModuleConcatenation bailout: Cannot concat with external "koa-router" (<- Module is not an ECMAScript module)
          ModuleConcatenation bailout: Cannot concat with external "lodash" (<- Module is not an ECMAScript module)
          ModuleConcatenation bailout: Cannot concat with external "mongoose" (<- Module is not an ECMAScript module)
          ModuleConcatenation bailout: Cannot concat with external "request" (<- Module is not an ECMAScript module)
          ModuleConcatenation bailout: Cannot concat with external "shortid" (<- Module is not an ECMAScript module)
          single entry ./src/server/index.js [16] multi ./src/server/index.js main[0]
          | ./src/server/index.js 298 bytes [depth 1] [built]
          |     [no exports]
          |     ModuleConcatenation bailout: Module is referenced from these modules with unsupported syntax: multi ./src/server/index.js (referenced with single entry)
          |     single entry ./src/server/index.js [16] multi ./src/server/index.js main[0]
          | ./src/server/server.js 2.22 KiB [depth 2] [built]
          |     [exports: default]
          |     [all exports used]
          |     harmony side effect evaluation ./server  ./src/server/index.js 1:0-27
          |     harmony import specifier ./server  ./src/server/index.js 5:0-3
          | ./src/server/jobs/invoices.js 5.41 KiB [depth 2] [built]
          |     [exports: checkPendingInvoices, runLastBlockJob]
          |     [only some exports used: runLastBlockJob]
          |     harmony side effect evaluation ./jobs/invoices  ./src/server/index.js 3:0-50
          |     harmony import specifier ./jobs/invoices  ./src/server/index.js 8:0-15
          | ./src/server/jobs/callbacks.js 3.13 KiB [depth 2] [built]
          |     [exports: checkPendingNotifications, runCallbackJob]
          |     [only some exports used: runCallbackJob]
          |     harmony side effect evaluation ./jobs/callbacks  ./src/server/index.js 4:0-50
          |     harmony import specifier ./jobs/callbacks  ./src/server/index.js 9:0-14
          | ./src/server/invoice/manager.js 4.54 KiB [depth 3] [built]
          |     [exports: default]
          |     [all exports used]
          |     harmony side effect evaluation ../invoice/manager  ./src/server/jobs/callbacks.js 4:0-42
          |     harmony side effect evaluation ./invoice/manager  ./src/server/api_v1.js 4:0-41
          |     harmony side effect evaluation ../invoice/manager  ./src/server/jobs/invoices.js 5:0-42
          |     harmony import specifier ../invoice/manager  ./src/server/jobs/callbacks.js 21:19-27
          |     harmony import specifier ../invoice/manager  ./src/server/jobs/invoices.js 23:19-27
          |     harmony import specifier ./invoice/manager  ./src/server/api_v1.js 39:19-27
          |     harmony import specifier ../invoice/manager  ./src/server/jobs/callbacks.js 46:34-42
          |     harmony import specifier ./invoice/manager  ./src/server/api_v1.js 71:19-27
          |     harmony import specifier ../invoice/manager  ./src/server/jobs/invoices.js 80:25-33
          | ./src/server/api_v1.js 2.33 KiB [depth 3] [built]
          |     [exports: default]
          |     [all exports used]
          |     harmony side effect evaluation ./api_v1  ./src/server/server.js 11:0-27
          |     harmony import specifier ./api_v1  ./src/server/server.js 15:64-67
          | ./src/contracts/index.js 267 bytes [depth 3] [built]
          |     [exports: DAI, ethProvider]
          |     [all exports used]
          |     harmony side effect evaluation ../../contracts  ./src/server/jobs/invoices.js 4:0-51
          |     harmony import specifier ../../contracts  ./src/server/jobs/invoices.js 151:21-32
          |     harmony import specifier ../../contracts  ./src/server/jobs/invoices.js 161:26-37
          |     harmony import specifier ../../contracts  ./src/server/jobs/invoices.js 163:42-45
          |     harmony import specifier ../../contracts  ./src/server/jobs/invoices.js 163:47-58
          | ./src/common/utils.js 223 bytes [depth 3] [built]
          |     [exports: fromBN, toBN]
          |     [only some exports used: fromBN]
          |     harmony side effect evaluation ../../common/utils  ./src/server/jobs/invoices.js 6:0-44
          |     harmony import specifier ../../common/utils  ./src/server/jobs/invoices.js 52:22-28
          | ./src/server/invoice/models.js 1.24 KiB [depth 4] [built]
          |     [exports: DepositAddress, Invoice]
          |     [all exports used]
          |     harmony side effect evaluation ./models  ./src/server/invoice/manager.js 5:0-51
          |     harmony import specifier ./models  ./src/server/invoice/manager.js 32:23-37
          |     harmony import specifier ./models  ./src/server/invoice/manager.js 45:23-37
          |     harmony import specifier ./models  ./src/server/invoice/manager.js 61:49-56
          |     harmony import specifier ./models  ./src/server/invoice/manager.js 62:25-39
          |     harmony import specifier ./models  ./src/server/invoice/manager.js 90:13-20
          |     harmony import specifier ./models  ./src/server/invoice/manager.js 100:13-20
          |     harmony import specifier ./models  ./src/server/invoice/manager.js 107:13-20
          |     harmony import specifier ./models  ./src/server/invoice/manager.js 116:13-20
          |     harmony import specifier ./models  ./src/server/invoice/manager.js 130:50-57
          |     harmony import specifier ./models  ./src/server/invoice/manager.js 153:13-27
          |     harmony import specifier ./models  ./src/server/invoice/manager.js 162:13-27
          | ./src/common/validators.js 1.06 KiB [depth 4] [built]
          |     [exports: validateInvoiceForm]
          |     [all exports used]
          |     harmony side effect evaluation ../common/validators  ./src/server/api_v1.js 5:0-59
          |     harmony import specifier ../common/validators  ./src/server/api_v1.js 24:21-40
          | ./src/contracts/eth_provider.js 289 bytes [depth 4] [built]
          |     [exports: default]
          |     [all exports used]
          |     harmony side effect evaluation ./eth_provider  ./src/contracts/index.js 1:0-41
          |     harmony import specifier ./eth_provider  ./src/contracts/index.js 5:86-97
          |     harmony export imported specifier ./eth_provider  ./src/contracts/index.js 6:0-23
          | ./src/server/connector.js 485 bytes [depth 5] [built]
          |     [exports: default]
          |     [all exports used]
          |     harmony side effect evaluation ../connector  ./src/server/invoice/models.js 1:0-30
          |     harmony import specifier ../connector  ./src/server/invoice/models.js 10:31-33
          |     harmony import specifier ../connector  ./src/server/invoice/models.js 21:24-26
          |     harmony import specifier ../connector  ./src/server/invoice/models.js 36:10-12
          |     harmony import specifier ../connector  ./src/server/invoice/models.js 72:28-30
          |     harmony import specifier ../connector  ./src/server/invoice/models.js 73:21-23

    LOG from webpack.buildChunkGraph.visitModules
    <t> prepare: 0.930023ms
    <t> visiting: 0.625132ms


   ╭────────────────────────────────────────────────────────────────╮
   │                                                                │
   │      New patch version of npm available! 6.13.4 → 6.13.6       │
   │   Changelog: https://github.com/npm/cli/releases/tag/v6.13.6   │
   │               Run npm install -g npm to update!                │
   │                                                                │
   ╰────────────────────────────────────────────────────────────────╯

ubuntu@ubuntu:~/daipay$ npm start

> daipay@1.0.1 start /home/ubuntu/daipay
> NODE_ENV=production node ./dist/server

connecting to mongodb://localhost/daipay
WARNING: NODE_ENV value of 'production' did not match any deployment config file names.
WARNING: See https://github.com/lorenwest/node-config/wiki/Strict-Mode
(node:4994) DeprecationWarning: current Server Discovery and Monitoring engine is deprecated, and will be removed in a future version. To use the new Server Discover and Monitoring engine, pass option { useUnifiedTopology: true } to the MongoClient constructor.
Listening on port 8000
(node:4994) DeprecationWarning: collection.ensureIndex is deprecated. Use createIndexes instead.
checkPendingInvoices: found 1 invoices
updating invoice #r5RXLFbf { state: 'expired' }
(node:4994) DeprecationWarning: collection.update is deprecated. Use updateOne, updateMany, or bulkWrite instead.
checkPendingInvoices: found 0 invoices
checkPendingInvoices: found 0 invoices
checkPendingInvoices: found 0 invoices
```

And it shows `Not Found` on http://127.0.0.1:8000

**Comment by defisapiens** [comment]
In production mode you need to use an external web server, such as nginx, to serve html content.
